core/src/main/java/org/mskcc/cbio/portal/util/GeneticProfileReader.java

```diff
@@ -275,4 +276,10 @@ public static String loadGenePanelInformation(File file) throws Exception {
         properties.load(new FileInputStream(file));
         return properties.getProperty("gene_panel");
     }
+
+	public static String loadFilteredMutations(File file) throws Exception {
```

Comment:
I would call this getVariantClassificationFilter. Please document here what the method does (gets a setting) and briefly what the purpose of this variant_classification_filter setting is .